benchmark_utils/utils.py

```diff
@@ -17,7 +17,7 @@ def _download_file_with_progress(url, filename):
     # is sometimes so long that it seems broken
     response = requests.get(url, stream=True, timeout=5)
     total_size_in_bytes = int(response.headers.get('content-length', 0))
-    block_size = 16384 #16 Kilobyte
+    block_size = 16384  # 16 Kilobyte
```

Comment:
are you using a linter that formats code?

Reply:
yes, the default one in vs code I assume?